test/testPartyAttachments.js

```diff
@@ -875,6 +875,93 @@ exports = module.exports = function (httpClient, type) {
         assert.equal(responseGetCopyAtt2.status, 200);
       });
 
+      it("copy attachments should work without 'file' property", async () => {
```

Comment:
This should probably be the code for the test above since the `file` property is not needed for /copy.

Reply:
file can be used in /copy to provide a new name for the copied attachment